ledger/txtail.go

```diff
@@ -101,32 +119,31 @@ func (t *txTail) committedUpTo(rnd basics.Round) basics.Round {
 			delete(t.recent, r)
 		}
 	}
+	for lv := range t.lastValid {
+		if lv < rnd {
+			delete(t.lastValid, lv)
+		}
+	}
```

Comment:
nit: you could avoid this loop by storing the lowest value in the lastValid map keys. If the difference is one, you're guaranteed that there is only a single item that need to be deleted.

Reply:
That's a good point, and keeping the last value committed to the ledger may be useful for other things as well.